features/environment.py

```diff
@@ -344,6 +345,12 @@ def backup(self, dest=os.path.join('data', 'basebackup')):
                         '--datadir=' + os.path.join(self._work_directory, dest),
                         '--dbname=' + self.backup_source])
 
+    def read_patroni_log(self, type):
+        try:
+            with open(str(os.path.join(self._output_dir or '', self._name + ".log"))) as f:
+                return [line for line in f.readlines() if line[24:24 + len(type)] == type]
+        except IOError:
+            return []
 
```

Comment:
flake8 expects 2 blank lines
```suggestion


```

Reply:
Got it, thanks! One thing I've noticed is that I seem to be passing `flake8` locally (although I agree the above is clearly off). When I run both `python setup.py flake8` and `tox -e lint` they pass with no errors. I just reread the contributing docs, and am not sure what might be going wrong here. Any ideas? (Don't want to bother with these kind of fixes in the future.)

I do see where the linting errors showed up in Codacy, so at the very least I'll be sure to look there going forward.